www/docs/src/content/docs/how-it-works/index.mdx

```diff
@@ -0,0 +1,77 @@
+
+## How does it work?
+
+AstroStudioCMS is devided into multiple parts:
+
+- **StudioCMS Core**: The core of StudioCMS is the primary integration that hooks into your Astro project and provides the ability to interact with the AstroStudio (AstroDB) integration with a prebuilt front-end.
+- **StudioCMS: Dashboard**: The StudioCMS dashboard is the primary web-based interface for managing your AstroStudioCMS project. It provides a user-friendly interface for creating, editing, and deleting content for your Astro project.
+- **StudioCMS: ImageHandler**: This integration provides a `<CustomImage />` Astro component that is used to handle remote images and optimize them for your Astro project in a SSR environment.
+- **StudioCMS: Robots-TXT**: This integration automates the creation of a robots.txt file for your Astro project and AstroStudioCMS.
+
+## Breakdown
+
+### StudioCMS Core
+
+#### Overview
+
+StudioCMS Core is an integration designed for Astro projects, enhancing them with a comprehensive CMS solution. This integration registers StudioCMS as the first plugin, sets up database configurations, watches for configuration changes, and adds various functionalities including virtual imports, logging, and default frontend routes.
+
+#### Features
+- **@astrojs/db**: The StudioCMS Core integration uses the `@astrojs/db` integration to connect to the AstroDB/AstroStudio.
+- **Virtual Imports**: The StudioCMS Core integration adds virtual imports to your Astro project to allow for dynamic content.
+- **Logging**: The StudioCMS Core integration logs all changes to the AstroDB/AstroStudio and can be enabled or disabled through a config option.
+- **Default Frontend Routes**: The StudioCMS Core integration adds default frontend routes to your Astro project to allow for easy access to the StudioCMS dashboard.
+- **Plugin System**: The StudioCMS Core integration provides a plugin system that allows you to extend the functionality of StudioCMS.
+
+#### Core Virtual Modules
+
+- **`virtual:studiocms/config`**: The Primary virtual module that provides the resolved configuration for the StudioCMS integration ecosystem.
+- **`virtual:studiocms/version`**: The Virtual module that provides the current version of the StudioCMS integration ecosystem.
+- **`virtual:studiocms/_nav`**: The Virtual module that provides the navigation mappings for the userfacing Front-end.
+- **`virtual:studiocms/astromdremarkConfig`**: The Virtual module that provides the configuration for the Astro markdown support.
+- **`studiocms:components`**: The Virtual module that provides the front-end exported components for StudioCMS.
+- **`studiocms:helpers`**: The Virtual module that provides the front-end exported helper functions for StudioCMS.
```

Comment:
```suggestion
- **`studiocms:helpers`**: Provides the front-end exported helper functions for StudioCMS.
```

It's also 100% worth adding a link to another config page which details what these helpers are.

Actually might be worth adding links to all these bullet points tbh...

Reply:
That is the plan... this is only the first of the planned pages?